### ISSUE TITLE
Do not install docs tools on Python 2.6

### ISSUE THREAD

**Issue by Glutexo** [RedHatInsights/insights-core#1387]
With the tools currently used it is not possible to generate documentation with Python 2.6. [nbsphinx](
https://pypi.org/project/nbsphinx/) it is not compatible with this legacy version of Python and does not even install. This was found out in #1385.

The build tools already take that into account: [build.sh](https://github.com/RedHatInsights/insights-core/blob/8510d6c072bed1a0c9823006ae3c6c9ada30791c/build.sh#L11), [shippable.yml](https://github.com/RedHatInsights/insights-core/blob/372f9228f37c7e118ffc70cdcc4c8fd8b9ddc6cd/shippable.yml#L17). Currently we can add a condition to the [docs packages](https://github.com/RedHatInsights/insights-core/pull/1369/files#diff-2eeaed663bd0d25b7e608891384b7298R51) not to install on Python ≤2.7. This would unblock `pip install -e .[develop]` before #1369 is merged. It might be also nice to print some warning.

After splitting the [develop bundle](https://github.com/RedHatInsights/insights-core/blob/372f9228f37c7e118ffc70cdcc4c8fd8b9ddc6cd/setup.py#L42) I think it would be good to check for the Python version and refuse to install the [docs bundle](https://github.com/RedHatInsights/insights-core/pull/1369/files#diff-2eeaed663bd0d25b7e608891384b7298R51) completely.

For the former it should be possible to use [Environment markers](https://www.python.org/dev/peps/pep-0508/#environment-markers). Unfortunately I am currently not sure how to do the warning or how to prevent install when running `pip install -e .`, so no specific suggestion for now.

It would be also nice to mention somewhere in the readme and/or another places where Python 2.6 support is mentioned, that this does not apply for docs.

**Comment by Glutexo** [comment]
Thanks to #1369 this is no longer relevant. Closing.
